pkg/policyreport/policyreport.go

```diff
@@ -225,3 +215,8 @@ func isDeletedPolicyKey(key string) (policyName, ruleName string, isDelete bool)
 
 	return "", "", false
 }
+
+func mapToStruct(in, out interface{}) error {
```

Comment:
Is `mapToStruct` an accurate name here? I don't have a better suggestion unfortunately :sweat_smile: